default/CMakeLists.txt

```diff
@@ -4,4 +4,4 @@ project(cmake_wrapper)
 include(conanbuildinfo.cmake)
 conan_basic_setup()
 
-add_subdirectory("source_subfolder")
+add_subdirectory("${CMAKE_BINARY_DIR}/source_subfolder")
```

Comment:
will this work for header-only packages, where we use no_copy_source?

Reply:
Oh, I am wrong!
This should have been added to `conanbuildinfo.cmake`!!
Sorry!!!!